src/pyaro/timeseries/Filter.py

```diff
@@ -819,3 +820,37 @@ def filter_data_idx(self, data: Data, stations: dict[str, Station], variables: s
             for minmax in self._minmax:
                 idx |= (minmax[0] <= data_resolution) & (data_resolution <= minmax[1])
         return idx
+
+@registered_filter
+class AltitudeFilter(StationReductionFilter):
+    """
+    Filter which filters stations based on their altitude. Can be used to filter for a
+    minimum and/or maximum altitude.
```

Comment:
Please document if min/max is included or excluded.
Please document what happens with stations with no altitude. (Preferrably removed if this filter is used?)